contracts/InterruptibleInverseVester.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.3;
+import "./InverseVester.sol";
+
+/**
+ * @title Interruptible inverse token vesting contract
+ * @author Inverse Finance
+ * @notice Vesting contract which can be interrupted by governance
+ */
+contract InterruptibleInverseVester is InverseVester {
+    address public governance;
+
+    /**
+     * @dev Prevents non governance from calling a method
+     */
+    modifier onlyGovernance() {
```

Comment:
What do you think of having "timelock" in this name in some form, e.g. `onlyGovernanceTimelock`/`onlyTimelock`? I don't expect the governance contract itself will be calling the vesting contracts. I would expect a proposal to occur on governance, which queues actions on a timelock, which in turn calls the vesting contract,